src/test/java/io/github/azagniotov/metrics/reporter/cloudwatch/CloudWatchReporterTest.java

```diff
@@ -598,11 +599,11 @@ private void buildReportWithSleep(final CloudWatchReporter.Builder cloudWatchRep
      *
      * @throws NoSuchFieldException
      * @throws IllegalAccessException
-     * @see Meter#tickIfNecessary()
+     * @see ExponentialMovingAverages#tickIfNecessary()
      * @see MetricDatum#getDimensions()
      */
     private static void reduceMeterDefaultTickInterval() throws NoSuchFieldException, IllegalAccessException {
```

Comment:
@madanepuri can you please provide some context about why this function has to be updated? Also, if it must to be updated, can you also update its name

Reply:
Also, please update the docs as you are changing the function behavior and this needs to be clearly communicated

Reply:
@azagniotov  `TICK_INTERVAL` field is moved to ExponentialMovingAverages class now. Previously it was in Meter class. Updated the function names. Please review. Thanks